tests/unit/plugins/service/test_enricher.py

```diff
@@ -127,20 +162,52 @@ def gce_enricher(config, metrics, mock_http_client):
     (exceptions.GCPHTTPError('404 error'), 0, 1, 'GCPHTTPError: 404 error'),
     ([{}] * 5, 4, 5, 'KeyError: \'networkInterfaces\'')])
 @pytest.mark.asyncio
-async def test_handle_message_event_msg_failures(
-        mocker, gevent_msg, mock_async_sleep, gce_enricher, caplog, response,
-        sleep_calls, logs_logged, err_msg):
-    """Raise error while enriching event message."""
+async def test_handle_message_event_msg_additions_failures(
+        mocker, additions_gevent_msg, mock_async_sleep, gce_enricher, caplog,
+        response, sleep_calls, logs_logged, err_msg):
+    """Raise error while enriching event message with additions."""
     gce_enricher._http_client._get_json_mock.side_effect = response
-    gevent_msg.phase = 'enrich'
```

Comment:
Are you sure this isn't required?

Reply:
The test seems to simply set that value and then check that the value is set. The assertion does not pass without the `gevent_msg.phase = 'enrich'` assignment.

Reply:
I did more digging.  The consumer code is supposed to update the gevent_msg with the next phase.  I think the idea was to make sure the input to the test is as real as possible.  I'm not sure if we actually need the assert in that case, though.

However:  It looks like that update is now handled by the router, and should be removed from the consumer?  @econchick ?

Reply:
yeah, that should only happen in the router now

Reply:
so this line isn't needed anymore if it happens in the router? https://github.com/spotify/gordon-gcp/blob/d3bef0bfbc1943d21fb8896fcb99d0df7cb6e708/src/gordon_gcp/plugins/service/enricher.py#L232

Reply:
No, that line is about history, not phase.  What needs to be removed is update_phase in the consumer.

For the enricher, I'd say leave the assignment line and remove the assert.

Reply:
Well, that just adds a log message... and it looks like that has not been fully integrated into the router yet, which actually should be fixed. Router just calls update phase and it does add a log message but it's just adding a bunch of messages with [plugin=None](https://github.com/spotify/gordon-gcp/blob/develop/src/gordon_gcp/plugins/service/event_consumer.py#L95).  In summary, I think you could remove it from enricher, but you do, you should also be making the logging fix to event_msg (in event_consumer), so that it logs something meaningful.

Reply:
:+1:

Can one of y'all make the story/issue about the `update_phase` + router stuff? I don't have enough context here.

Reply:
I can.

Reply:
I would leave this line in meanwhile.

Reply:
It's still in the code, `gevent_msg` has just been renamed to `additions_gevent_msg`.